veggieseasons/analysis_options.yaml

```diff
@@ -0,0 +1,9 @@
+include: package:pedantic/analysis_options.yaml
+
+linter:
```

Comment:
Consider adding:

analyzer:
  strong-mode:
    implicit-casts: false

https://dart.dev/guides/language/analysis-options#enabling-additional-type-checks

Reply:
Done. This has caused the most actual changes to the code, as it turns out.